src/UI/Document.elm

```diff
@@ -560,10 +578,18 @@ toBrowserDocument cfg pageItem (Document model) =
                 else
                     case model.sidebarStyle of
                         SidebarPersistent ->
-                            SideBar.desktopPersistent cfg contentBody model.menu
+                            if model.hasLegacySidebar then
+                                LegacySideBar.desktopPersistent cfg contentBody model.menu
+
+                            else
+                                SideBar.desktopPersistent cfg contentBody model.menu
 
                         SidebarNonPersistent ->
-                            SideBar.desktopNonPersistent cfg contentBody model.menu
+                            if model.hasLegacySidebar then
+                                Element.none
```

Comment:
This way we have an `Element.none`,  is this on purpose?

If not, then we should make legacy always be persistent...